sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -976,13 +976,6 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
       sql("insert into t select false, default")
       checkAnswer(spark.table("t"), Row(false, 42L))
     }
-    // There is a complex query plan in the SELECT query in the INSERT INTO statement.
-    withTable("t") {
-      sql("create table t(i boolean default false, s bigint default 42) using parquet")
-      sql("insert into t select col, count(*) from values (default, default) " +
```

Comment:
So this case becomes invalid after refactoring?

Reply:
The test case updated since the set of allowed operators above the `INSERT INTO ... VALUES` list changed. But it is fine to keep this a pure refactoring PR, I added `Aggregate` to the allowed operators and now all the test are exactly the same now.